package.json

```
@@ -9,7 +9,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "format": "prettier --write '{app,components,lib}/**/*.{ts,tsx,scss}'",
+    "type-check": "tsc --noEmit",
+    "format": "prettier --write '{app,components,lib}/**/*.{js,jsx,ts,tsx,json,css,scss,md}'",
```

You may want to add a dedicated script here like, so that you can run `npm run style-check` in the pipeline

```suggestion
    "format": "prettier --write '{app,components,lib}/**/*.{js,jsx,ts,tsx,json,css,scss,md}'",
    "style-check": "prettier --check '{app,components,lib}/**/*.{ts,tsx,js,jsx,json,css,scss,md}'",
```